src/wallet/sqlite.cpp

```diff
@@ -182,6 +182,14 @@ SQLiteBatch::SQLiteBatch(SQLiteDatabase& database, const char* mode)
 
 void SQLiteBatch::Close()
 {
+    // If m_db is in a transaction (i.e. not in autocommit mode), then abort the transaction in progress
+    if (m_database.m_db && sqlite3_get_autocommit(m_database.m_db) == 0) {
+        if (TxnAbort()) {
+            LogPrintf("SQLiteBatch: Batch closed unexpectedly without the transaction being explicitly commited or aborted\n");
```

Comment:
011fa1e2657d624593bb7f23f697a741f7b51577, typo:
```suggestion
            LogPrintf("SQLiteBatch: Batch closed unexpectedly without the transaction being explicitly committed or aborted\n");
```

Reply:
Done